examples/next/pages/ui/components/liveness/components/LivenessInlineResults.tsx

```diff
@@ -13,6 +13,10 @@ export default function LivenessInlineResults({
 }) {
   const { isLive, confidenceScore, auditImageBytes } = getLivenessResponse;
 
+  // Previously the rekognition backend passed the base64 string but from the api direclty you receive a byte array
+  var base64string = Buffer.from(
+    new Uint8Array(Object.values(auditImageBytes))
+  ).toString('base64');
```

Comment:
This can be done in the lambda itself (to convert bytearray to str).

Reply:
Will do in a follow up, want to get this PR out!